README.md

```diff
@@ -46,9 +46,88 @@ walkers in 5 containers in YARN using the Image.xml module.
 
         ./bin/accumulo-testing rw-yarn 5 Image.xml
 
-This command will create an application in YARN and exit when all containers for the test have started.
-While its running, you can view logs for each random walker using the YARN resource manager. The YARN
-application can be killed at any time using the YARN resource manager or command line tool.
+This command will create an application in YARN and exit when all containers for the test have
+started. While its running, you can view logs for each random walker using the YARN resource manager.
+The YARN application can be killed at any time using the YARN resource manager or command line tool.
+
+## Continuous Ingest & Query
+
+The Continuous Ingest test consists of several applications that place a continuous query and ingest
+load on Accumulo to see if it breaks.
```

Comment:
I think it would be useful to state that CI creates lots of large linked list and the test suite can look for holes in these linked list to see if Accumulo lost data.  Could pull from [goraci]'s background section.

[goraci]: https://github.com/keith-turner/goraci